cmd/porter/bundle.go

```diff
@@ -154,6 +158,8 @@ The first argument is the name of the claim to uninstall. The claim name default
 		"Define an individual parameter in the form NAME=VALUE. Overrides parameters set with the same name using --param-file. May be specified multiple times.")
 	f.StringSliceVarP(&opts.CredentialIdentifiers, "cred", "c", nil,
 		"Credential to use when uninstalling the bundle. May be either a named set of credentials or a filepath, and specified multiple times.")
+	f.StringVarP(&opts.Driver, "driver", "d", "docker",
+		"Specify a driver to use. Defaults to 'docker'.")
```

Comment:
This should list the allowed drivers, e.g. "Allowed values: docker, debug". 

Also, it's not explained what the "debug" driver is, or what a driver is. Maybe not here, feel free to use your judgement, but somewhere in this helptext. Right now we have a new example that says "--driver debug" and it's really not clear what that does. We could either add text to the example to explain, we could put a paragraph into the `Long` to explain drivers, or explain in the flag text.

Reply:
Updated; let me know how it reads/looks to you.  Thanks!